sdk/identity/azure-identity-broker/README.md

```diff
@@ -4,8 +4,10 @@ This package extends the [Azure Identity][azure_identity] library by providing s
 
 | Broker                                    | Minimum package version |
 |-------------------------------------------|-------------------------|
-| [Company Portal][company_portal] on macOS | 1.3.0b1                 |
 | Web Account Manager (WAM) on Windows 10+  | 1.0.0                   |
+| [Company Portal][company_portal] on macOS | 1.3.0b1                 |
+| Web Account Manager (WAM) on WSL          | 1.3.0b2                 |
```

Comment:
Should we specify the minimum version of WSL needed for this to work? Seems like 2.4.13+ is needed based on discussion here: https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/766/files#r2019328210